python/sidecar/http.py

```diff
@@ -1,24 +1,13 @@
 from dotenv import load_dotenv
 from fastapi import FastAPI
-
-import sidecar
-from sidecar.typing import (
-    ChatRequest,
-    ChatResponse,
-    DeleteRequest,
-    DeleteStatusResponse,
-    IngestRequest,
-    IngestResponse,
-    IngestStatusResponse,
-    ReferenceUpdate,
-    RewriteRequest,
-    RewriteResponse,
-    SearchRequest,
-    SearchResponse,
-    TextCompletionRequest,
-    TextCompletionResponse,
-    UpdateStatusResponse,
-)
+from sidecar import chat, ingest, rewrite, search, storage
+from sidecar.typing import (ChatRequest, ChatResponse, DeleteRequest,
+                            DeleteStatusResponse, IngestRequest,
+                            IngestResponse, IngestStatusResponse,
+                            ReferenceUpdate, RewriteRequest, RewriteResponse,
+                            SearchRequest, SearchResponse,
+                            TextCompletionRequest, TextCompletionResponse,
+                            UpdateStatusResponse)
```

Comment:
Can we leave the formatting here? I suspect prettier is doing something to Python files but I haven't been able to figure out how to stop it.